app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/ProgressBar.kt

```diff
@@ -0,0 +1,185 @@
+package com.hedvig.android.design.system.hedvig
+
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.DrawScope
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.semantics.ProgressBarRangeInfo
+import androidx.compose.ui.semantics.progressBarRangeInfo
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.ProgressDefaults.arcSweepAngle
+import com.hedvig.android.design.system.hedvig.ProgressDefaults.circularBaseWidth
+import com.hedvig.android.design.system.hedvig.ProgressDefaults.progressAnimationDuration
+import com.hedvig.android.design.system.hedvig.ProgressDefaults.stroke
+import com.hedvig.android.design.system.hedvig.ProgressDefaults.strokeCap
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.FillPrimary
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.SurfaceSecondary
+import kotlin.math.abs
+
+@Composable
+fun HedvigLinearProgressBar(modifier: Modifier = Modifier) {
+  val infiniteTransition = rememberInfiniteTransition(label = "rememberInfiniteTransition")
```

Comment:
Yeah I do not expect us to use this one, as we typically do not have things that we know have a determinate end and we also know the progress of it. If we have an indeterminate (one that keeps flowing and does not have a start -> finish progress) version of this perhaps we should use that one.